meetings/2016-11-9.md

```diff
@@ -0,0 +1,174 @@
+# Node Foundation CTC Meeting 2016-11-09
+
+## Links
+
+* **Audio Recording**: TBP
```

Comment:
Here it is: https://www.youtube.com/watch?v=xr8YKTnzu-4

It is titled wrong on YouTube. I'll fix it now.